src/locales/intl.test.js

```diff
@@ -0,0 +1,320 @@
+/* globals describe test expect beforeEach */
+import intl, {setIntlLocale} from './intl'
```

Comment:
We are standardizing against using unnamed exports. Please give `intl` a specific export name and change to something like

`import {intl, setIntLocale} from './intl/`